client/src/components/sidenav/index.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import '../../App.css';
+import './sidenav.css'
+
+class Sidenav extends Component {
+  render() {
+    const { thirdTap } = this.props;
+    return (
+      <div className="sidenav-container">
```

Comment:
You could use <ul> and <li> tags here
`<ul className='sidenav-container'>
      <li className='flights'>`
and so on.
Then in your css, instead of these:
`.flights, .add-flight, .update-flight {`
`.flights img, .add-flight img, .update-flight img {`
you could have
`li {`
`li img {`
And then, when you add more items to the sidenav, they will have the correct styles automatically, without having to add to those css selectors.

Reply:
But not using tag name in CSS file
Thanks
Done :+1: